sklearn/metrics/tests/test_common.py

```diff
@@ -1896,6 +1896,10 @@ def check_array_api_metric_pairwise(metric, array_namespace, device, dtype_name)
 
     metric_kwargs = {}
     if "dense_output" in signature(metric).parameters:
+        metric_kwargs["dense_output"] = False
+        check_array_api_metric(
+            metric, array_namespace, device, dtype_name, a_np=X_np, b_np=Y_np
+        )
```

Comment:
Hi all, just realised that we are not passing the `metric_kwargs` to the function (also in the function call below). I think it should be:

```python
check_array_api_metric(
             metric, array_namespace, device, dtype_name, a_np=X_np, b_np=Y_np, ** metric_kwargs
         )
```

should I make another PR?

Reply:
Ahrg! Yes please

Reply:
I apologize for missing this.

Reply:
I also missed it in the previous PR 😇

See fix here: https://github.com/scikit-learn/scikit-learn/pull/29045